### PR TITLE
Documentation edits

### LINKED ISSUE #8: Documentation tasks
- Name scripts to indicate some order of running as in https://github.com/claudiatebaldi/hectorcal/tree/master/data-raw/ESM_processing_code it is somewhat hard because there is python and R code that needs to be run for the different variables, but we will want to make sure that it is really clear what order scripts need to be run in and how any scripts that are dependent on another one are stated somewhere.
- Add read me to describe the contents of a directory does it contain a subdirectory full of data generated by some script? What does the processing scripts do?
- Add text to the beginning of each script to state the objective of the script it doesn’t need to be as formal as the header included here https://github.com/JGCRI/Emissions-MIP_Data/blob/main/code/summary_plots_diff.R, but something to indicate who authored the code, when it was written, and the overall objective would be really helpful. Feel free to reuse the description of the scripts from the read-me files.

### PR DESCRIPTION
**Pull request by leeyap**
Closes #8 

- Create `readme`
- Clean up code and general organization
- Write file headers
- Reorganize and rename .py and .R files to have a coherent order and run efficiently
- Create new directories - `./inputs` for csvs that need to be read into R, `./outputs` for final data frames out of the R scripts, `./scripts` for all R and Python files

@kdorheim please take a look when you get a chance and let me know if this format is okay, or if there are any other files that need editing. Thanks very much!

### CODE REVIEW

File: scripts/A3.co2.py [hunk 1]
```diff
@@ -1,6 +1,10 @@
-# Processing CMIP6 tas data using Pangeo
-# December 2021
-# Leeya Pressburger
+# ------------------------------------------------------------------------------
+# Program Name: A3.co2.py
+# Authors: Leeya Pressburger
+# Date Last Modified: February 2022
+# Program Purpose: Downloading CMIP6 `co2` data using Pangeo
```
Comment (kdorheim) [inline scripts/A3.co2.py]: I don't want to be too nit-picky here but it seems like this script does more than download the CMIP6 data, that it processes the co2 data and saves it at a csv. Could you add a note about how it is processing the data (global total/average is the data saved annual or monthly? are the results stored in a single csv file or is a csv file saved per model)? 

Those sorts of notes would be extremely helpful to have in the header.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+# Scripts
+The ./scripts directory contains both Python and R scripts for each variable. The Python scripts are used to access CMIP6 data using Pangeo. Outputs get saved as csv files, which are read into R for further processing and visualization. 
+
+Order of operations: Run the "A_" Python script, and then the corresponding "B_" R script/s. If present, RMarkdown files will have the prefix "C_."
+
+The ocean heat flux variables (hfls, hfss, rlds, rlus, rsds, rsus) all have individual Python files. 
+
+# Directories
+Each directory named for a variable contains raw csv output files for each variable. The files are generated by the Python scripts, leveraging Pangeo. The corresponding R scripts then use these csv files to perform data manipulations and calculations to yield final output files. These output files are also csv files, location in ./outputs
+
```
Comment (kdorheim) [inline README.md:15]: Are these output files cleaned up? (are outliers are removed, if there is an spp scenario is there a corresponding historic result? )

**Comment by kdorheim** [comment]
Potentially consider moving `hector_cmip6data/inputs/CMIP5_heat_flux_final.csv` to a directory like comparison data or misc data, or leave it where it is regardless please make it clear where this data came from since unlike the other csv files this wasn't generated by any of the scripts included in this repo. 

Something that is slightly unclear so the contents of the outputs directory are those the final formatted/cleaned up results ready for use? Also,  what does the frequency column refer to? That the original data is was monthly data that has now been aggregated to an annual value? Or are the contents of the file monthly? If that is the case the month information should be included. What is the File.x column? 

Ideally, the final csv output should be saved with a file name that is slightly more descriptive so that users can read the file name can have some expectation of what the csv file will contain (such as cmip6_annual_global_tas.csv and so on). 

The contents of the csv files look like, variable, experiment, units, value, model, ensemble, (year or time or data whatever is appropriate), (region or type etc if appropriate) every column should be named, and we don't want row indices or anything like that and we don't want columns that have duplicative information. 

Passing the argument `row.names = FALSE` to the  `write.csv` function will prevent the unnamed row with the numbers to be included in the csv file. 

Great job on processing complicated so much CMIP data. Setting up a reproducible workflow to process multidimensional data products is no easy task! Happy to meet to talk about the strategy behind setting up data in a particular manner to make clear and clean data files. After you done with the documentation & reog I will set up the tos processing data so that it matches you set up 👍

**Review by kdorheim — CHANGES_REQUESTED** [review]

**Comment by leeyap** [comment]
@kdorheim Thanks very much for your comments! I think I addressed them all in this round of commits. Would you mind taking another look when you get a chance and let me know if there's anything else you'd like me to tweak? I'll probably also take a second look after spending some time away.

Thanks 😃

**Review by kdorheim — APPROVED** [review]

**Comment by kdorheim** [comment]
Great job @leeyap!! Thank you for expanding on the documentation our future selves will be extremely grateful! After this is merged I'll open a PR to reorganize the tos data to make your set up with the exception being the `land-ocean-warming-ratio`, which was done pre-pangeo days but I'll clarify that in the read me 👍

**Comment by bpbond** [comment]
https://xkcd.com/1421/

**Pull request merged**